components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/APIExportUtil.java

```diff
@@ -111,10 +113,11 @@ public static void retrieveApiToExport(String archiveBasePath, API apiToReturn,
         try {
             registry = ServiceReferenceHolder.getInstance().getRegistryService().getGovernanceSystemRegistry(tenantId);
             //directory creation
-            CommonUtil.createDirectory(archivePath);
+            createDirectory(archivePath);
```

Comment:
Since we are calling static methods in a Util class, better to use the convention `CommonUtil.createDirectory(archivePath);`

Reply:
resolved